apps/frontend/src/pages/auth/libs/components/sign-up-form/sign-up-form.tsx

```diff
@@ -27,10 +27,12 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 	const handleFormSubmit = useCallback(
 		(event_: React.BaseSyntheticEvent): void => {
 			void handleSubmit((payload: UserSignUpFormDto) => {
-				const { confirmPassword, ...userData } = payload;
+				const { password } = payload;
```

Comment:
But in that case `UserSignUpFormDto` type is not correct, so it should be updated